hxckr-org/workshop-btc-txn-builder
# switch to bun and refactor test

This PR, move the dep. management to bun since that is what trigger test in test runner, it also refactors the test to run via bun test for simplicity. 


To test: 

- make sure you have bun install on your machine. 
- execute `bun test`
